### PR TITLE
Add SemaDB VST wrapper

### PR DESCRIPTION
**Pull request by nuric**
- **Description**: Adding vectorstore wrapper for [SemaDB](https://rapidapi.com/semafind-semadb/api/semadb).
- **Issue**: None
- **Dependencies**: None
- **Twitter handle**: semafind

Checks performed:
- [x] `make format`
- [x] `make lint`
- [x] `make test`
- [x] `make spell_check`
- [x] `make docs_build`

Documentation added:

- SemaDB vectorstore wrapper tutorial

### CODE REVIEW

File: libs/langchain/langchain/vectorstores/semadb.py [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+    """
+
+    HOST = "semadb.p.rapidapi.com"
+    BASE_URL = "https://" + HOST
+
+    def __init__(
+        self,
+        collection_name: str,
+        vector_size: int,
+        api_key: str,
```
Comment (baskaryan) [inline libs/langchain/langchain/vectorstores/semadb.py:34]: most integrations make it possible to also specify api keys via environment vars, thoughts on doing that here?
```python
from langchain.utils import get_from_env

class SemaDB(VectorStore):
    def __init__(
        ...
        *,
        api_key: Optional[str] = None,
        distance_strategy: DistanceStrategy = DistanceStrategy.EUCLIDEAN_DISTANCE,
    ) -> None:
        
        self.api_key = api_key or get_from_env("api_key", "SEMA_DB_API_KEY")
```

Comment (nuric) [reply libs/langchain/langchain/vectorstores/semadb.py:34]: Sure, that makes sense. I'll update the tutorial notebook too.

**Comment by nuric** [comment]
@baskaryan That's all updated with checks running locally.

**Comment by baskaryan** [comment]
looks great, thanks @nuric!

**Pull request merged**